test/include/util/storage_test_util.h

```diff
@@ -31,6 +33,8 @@ class StorageTestUtil {
   StorageTestUtil() = delete;
 
 #define TO_INT(p) reinterpret_cast<uintptr_t>(p)
+
+#define MAX_TEST_VARLEN_SIZE (5 * storage::VarlenEntry::InlineThreshold())
```

Comment:
Why such a small limit here (60 bytes)?

Also replace with `constexpr`.

Reply:
This was actually just 24 bytes until not too long ago. The [arrow pr](https://github.com/cmu-db/terrier/commit/271580a5d1a23f5d91ee19dadb32f307d7a19550#diff-4406bcd728f854fe2a85e018d4d06c9fR104) changed it to 60 bytes recently, so I don't have an answer.